scripts/spack/configs/blueos_3_ppc64le_ib_p9/spack.yaml

```diff
@@ -0,0 +1,288 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# Axom Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+spack:
+  config:
+    install_tree:
+      root: $spack/..
+      projections:
+        all: '{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+    misc_cache: $spack/../misc_cache
+    test_stage: $spack/../test_stage
+    build_stage::
+    - $spack/../build_stage
+
+  # Regular TPLs do not need views
+  view: false
+
+  compilers::
+  - compiler:
+      environment: {}
+      extra_rpaths: []
+      flags: 
+        # Fix for "undefined reference to `_gfortran_transfer_integer_write'"
+        ldlibs: -lgfortran
+      modules: []
+      operating_system: rhel7
+      paths:
+        cc:  /usr/tce/packages/gcc/gcc-8.3.1/bin/gcc
+        cxx: /usr/tce/packages/gcc/gcc-8.3.1/bin/g++
+        f77: /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
+        fc:  /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
+      spec: gcc@8.3.1
+      target: ppc64le
+  - compiler:
+      environment: {}
+      extra_rpaths: []
+      flags:
+        # Fix for "undefined reference to `_gfortran_transfer_integer_write'"
+        ldlibs: -lgfortran
+      modules: []
+      operating_system: rhel7
+      paths:
+        cc:  /usr/tce/packages/clang/clang-ibm-10.0.1-gcc-8.3.1/bin/clang
+        cxx: /usr/tce/packages/clang/clang-ibm-10.0.1-gcc-8.3.1/bin/clang++
+        f77: /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
+        fc:  /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
+      spec: clang@10.0.1
+      target: ppc64le
+  - compiler:
+      environment: {}
+      extra_rpaths: []
+      flags:
+        cflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
+        cxxflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
+        fflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
+      modules: []
+      operating_system: rhel7
+      paths:
+        cc: /usr/tce/packages/xl/xl-2022.08.19/bin/xlc
+        cxx: /usr/tce/packages/xl/xl-2022.08.19/bin/xlC
+        fc: /usr/tce/packages/xl/xl-2022.08.19/bin/xlf2003
+        f77: /usr/tce/packages/xl/xl-2022.08.19/bin/xlf
+      spec: xl@16.1.1.1
```

Comment:
Just a comment on the wonky versioning going on in this section.

Axom uses the compiler spec in the install path. Due to Spack reasons, we cannot have what we really want which is like `xl_16.1.1` and `xl_16.1.1_cuda`.

Because of this we add a `.1` (CPU build) and a `.2` (GPU build).

If you do not want/need that, then you can remove all these specs and the extra version number.

Reply:
I can get rid of it for now.  The xl build isn't working anyway.  Hypre wants the lapack without trailing underscores but mfem wants the trailing underscores... I did a patch before (see https://github.com/LLNL/Tribol/blob/develop/scripts/spack/packages/mfem/fix_name_mangling.patch), but I don't think it's worth keeping.